src/facade/facade_types.h

```diff
@@ -46,6 +49,44 @@ inline std::string_view ToSV(std::string_view slice) {
   return slice;
 }
 
+inline std::string_view ToSV(const std::string& slice) {
+  return slice;
+}
+
+inline std::string_view ArgS(CmdArgList args, size_t i) {
+  auto arg = args[i];
+  return {arg.data(), arg.size()};
+}
+
+constexpr auto kToSV = [](auto v) { return ToSV(v); };
+
+struct ArgRange {
+  template <typename T,
+            typename = typename std::enable_if_t<!std::is_base_of_v<ArgRange, std::decay_t<T>>>>
```

Comment:
I think a  better approach define move and copy constructors separately and you don't need this enable_if in this case

Reply:
Yep, seems simpler